### DISCUSSION TITLE
HTML link in component loads entire site

### DISCUSSION THREAD

**Discussion by davestewart** [vuejs/vitepress#4210] [Q&A] [answered]
Hello,

In my docs site, I have a [Card](https://github.com/spectre-org/spectre-docs/blob/main/.vitepress/theme/components/Card.vue) component which functions as a link. I [globally register](https://github.com/spectre-org/spectre-docs/blob/main/.vitepress/theme/index.ts#L34) it then use it on index pages:

- https://spectre-org.github.io/spectre-docs/docs/get-started/

However, the HTML `a` element inside seems to trigger a fresh load of the page, rather than working in SPA mode.

Here is the index page markup:

```md
# Get Started

Get started with the Spectre CSS library.

<div class="vp-raw docs-demo columns">
  <Card title="Installation"
        desc="How to install and use Spectre CSS"
        path="installation.html" />
  ...
</div>
```

If I add a straight markdown link in the page, the navigation works perfectly.

What am I missing to make this link work like it should?

Thanks.

**Comment by brc-dd** [comment, marked as answer]
https://github.com/vuejs/vitepress/issues/4112

It's because of vp-raw. vp-raw currently bypasses the SPA router as well.

**Reply by davestewart** [reply]
Aha! Thank you.

Now #4189 is fixed, I'll look to remove this.
